.editorconfig

```diff
@@ -13,3 +13,5 @@ trim_trailing_whitespace = false
 [{.travis.yml,npm-shrinkwrap.json,package.json}]
 indent_style = space
 indent_size = 2
+[Makefile]
+indent_style = tab
```

Comment:
Well, it is possible to use another character than tabs... but not really useful.

See [`.RECIPEPREFIX`](https://www.gnu.org/software/make/manual/make.html#Special-Variables)